fleet_notifications/incoming_call_endpoint.py

```
@@ -0,0 +1,64 @@
+    
+    try:
+        car_id = _allowed_incoming_phone_numbers[request.values['From']]
+        action_status = _car_action_api.get_car_action_states(car_id, last_n=1)[0].action_status
+
+        if action_status == CarActionStatus.PAUSED:
+            _car_action_api.unpause_car(car_id)
+            resp.say(f"Car succesfully unpaused.")
+        else:
+            _car_action_api.pause_car(car_id)
+            resp.say(f"Car succesfully paused.")
+    except:
```

Ruff (0.8.0)

Do not use bare `except`

(E722)